denorm/db/postgresql/triggers.py

```diff
@@ -132,34 +132,21 @@ def sql(self):
 
 
 class TriggerSet(base.TriggerSet):
-    def drop_atomic(self):
+    def cursor(self):
+        if not hasattr(self, '_cursor'):
+            cursor = super(TriggerSet, self).cursor()
+            cursor.execute("SELECT lanname FROM pg_catalog.pg_language WHERE lanname ='plpgsql'")
```

Comment:
This will now get called in the drop flow (it doesn't in existing code) but it should be harmless.